communication/marketing-team/role-handbooks/Tech-Lead.md

```diff
@@ -0,0 +1,55 @@
+# Tech lead Role Handbook
```

Comment:
nit: looks like title case is used for headers, the following suggestion keeps the pattern
```suggestion
# Tech Lead Role Handbook
```

Reply:
done